__tests__/integration.test.js

```diff
@@ -149,6 +150,61 @@ describe("/api/articles/:article_id/comments", () => {
                 expect(response.body.msg).toBe("Invalid Request");
             });
     });
+    test("POST:201 Should insert the new comment into the database and send the object back to the client", () => {
+        return request(app)
+            .post("/api/articles/2/comments")
+            .send({
+                username: "rogersop",
+                body: "This is a test comment...",
+            })
+            .expect(201)
+            .then((response) => {
+                const comment = response.body.comment;
+                expect(Object.keys(comment).length).toBe(6);
+                expect(typeof comment.comment_id).toBe("number");
```

Comment:
could be more specific in some of your assertions here i.e. with the value of comment id - as long as you're reseeding before each test we should be able to expect the id exactly which I'd say there's an argument for checking!